source/content/plugins-known-issues.md

```diff
@@ -63,15 +63,15 @@ The following is a list of plugins that assume write access, and the specific fi
 
 ### Define FS_METHOD
 
-By default, WordPress tests each directory before uploading a file by writing a small temporary file. There are several plugins and themes that have issues on Pantheon due to this write access test. By defining the `FS_METHOD` as `direct` in `wp-config.php` above the line `/* That's all, stop editing! Happy Pressing. */`, we can easily avoid these issues and skip the test of writing a small file. The successful write of that temporary file results in `direct` in the end, so this makes operations slightly faster as well.
+By default, WordPress tests each directory before uploading a file by writing a small temporary file. Some plugins and themes may have issues on the Pantheon platform due to this write access test. You can avoid these issues (and skip the test of writing a small file) by defining the `FS_METHOD` as `direct` in the `wp-config.php` file above the line `/* That's all, stop editing! Happy Pressing. */`. The successful write of the temporary file results in the addition of `direct` at the end of the file, which allows operations to run slightly faster. To resolve the issue, configure the `wp-config.php` to resemble the following code sample:
```

Comment:
@joa-pan one change here, in the edit we ended up with: 

> The successful write of the temporary file results in the addition of `direct` at the end of the file, 

This isn't quite right, what I meant was that when WordPress is testing by writing those temporary files, it is trying to determine if it can use "direct", and when it successfully does write that file it returns "direct". So setting it ahead of time gets you the same result. It isn't really at the end of the file but the result of the function in the file class.

Reply:
Ok! Thanks for that explanation @jms-pantheon. I'll update accordingly 🎉

Reply:
Yes looks good!